### PR TITLE
Feature/komarch 18 import images

### PR DESCRIPTION
**Pull request by yanchith**
As discussed on call, this task was retrofitted into just the image downloader and we'll implement informed, metadata driven image processing and import in an upcoming task.

The PR is still a draft, because:

- [x] I need to find out what are the `tmp` directory conventions for Laravel, and download the images there,
- [x] I need to investigate why a `try/catch` block does not catch this `FileNotFound` exception: "File not found at path: wp-content/uploads/2016/07/VVP-POPRAD-GrafickÃƒÂ¡-Ã„asÃ…Â¥_Strana_1-150x150.jpg".

I expect to resolve both issues soon, but this can meanwhile be reviewed.

### CODE REVIEW

File: app/Console/Commands/ImportWp.php [hunk 1]
```diff
@@ -9,28 +9,69 @@
...
+                    $this->info('Copying file: ' . $file);
+                    try {
+                        $contents = $wordpressFs->get($file);
+                        Storage::put($file, $contents);
+
+                        $copiedCount++;
+                    } catch (Exception $e) {
+                        // XXX: This catch doesn't catch FileNotFoundException
+                        $this->warn('Failed to copy file: ' . $file);
+                    }
```
Comment (yanchith) [inline app/Console/Commands/ImportWp.php:61]: This is the `try/catch` block that doesn't catch , btw... thoughts?

**Comment by yanchith** [comment]
Okay, both issues mentioned above are now resolved.

File: app/Console/Commands/ImportWp.php [hunk 2]
```diff
@@ -9,28 +9,111 @@
...
         if (!$this->confirm('⚠️ This will truncate all current posts! Do you wish to continue?')) {
             return $this->info('OK. Bye.');
         }
 
+        if ($this->option('download-images')) {
+            $this->info('Downloading images to temporary directory');
+            $this->info('Connecting to wordpress filesystem');
+
+            $wordpressFs = Storage::disk('wordpress');
+            $tempDir = sys_get_temp_dir();
```
Comment (igor-kamil) [inline app/Console/Commands/ImportWp.php:36]: we used [spatie/temporary-directory](https://github.com/spatie/temporary-directory) in regarch.  IJS. 
https://github.com/SlovakNationalGallery/register-architektury/blob/master/app/Jobs/ProcessBuildingImage.php

**Review by igor-kamil — APPROVED** [review]
good for now. more after analysis.

File: app/Console/Commands/ImportWp.php (hunk 2 above)
Comment (yanchith) [reply app/Console/Commands/ImportWp.php:36]: Thanks! I'll keep this in mind. We can change the current implementation to spatie/temporary-directory the next time we touch this code, although I looked at its source directory, and it is a very small single-file lib, so maybe there is not that much benefit there.

**Pull request merged**